plugins/woocommerce/bin/lint-branch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Lint branch
+#
+# Runs phpcs-changed, comparing the current branch to its "base" or "parent" branch.
+# The base branch defaults to trunk, but another branch name can be specified as an
+# optional positional argument.
+#
+# Example:
+# ./lint-branch.sh base-branch
+
+baseBranch=${1:-"trunk"}
+
+changedFiles=$(git diff $(git merge-base HEAD $baseBranch) --relative --name-only -- '*.php')
+
+# Only complete this if changed files are detected.
+[ -z $changedFiles ] || composer exec phpcs-changed -- -s --git --git-base $baseBranch $changedFiles
```

Comment:
Enclosing `$changedFiles` in double quotes allows the script to work ok when multiple files have been modified. See https://github.com/woocommerce/woocommerce/pull/38085#issuecomment-1547913636.

Reply:
Oops, from your other comment I got fixated on the brackets thing and missed that this was about quotes 🤦